templates/site/js/load-node-content.js

```diff
@@ -29,10 +29,27 @@ function buildNodeDescription(name, data, node_index, row_id_suffix) {
                     img_alt = link.alt;
                 }
                 break
+
         }
     })
+
+    if(window.location.href.includes("ide.html")){
```

Comment:
indentations in this block are very inconsistent. Please be consistent with indentations and blank lines

Reply:
`includes` will find any substring in the href and so it might return true unexpectedly. 

Also, this strategy of having a special condition for a specific page is not very portable and will eventually require us to have lots of if conditions, one for each page. 

A better and more portable strategy is to have an optional function that can be defined on each page (ide.html, datacatalog.html, etc.) that can be called to set the correct href link.  For an example on how to do this, see how we're handing the `nodeFilter` function (at the bottom of this file).